### PR TITLE
new Player functionality

### PR DESCRIPTION
**Pull request by havecookies**
1. Added properties to access postion and spritePath.
2. Added MoveTo method to change the position variable in Player.
3. Added new documentation.
4. position, coins, and arrows are now ushorts instead of uints.

### CODE REVIEW

File: Core/WumpusCore/Player/Player.cs [hunk 1]
```diff
@@ -7,33 +7,60 @@ public class Player
...
-            coins = 0;
-            arrows = 0;
-
-            spritePath = "";
-
             if (!File.Exists(spritePath))
             {
                 throw new FileNotFoundException();
             }
+            this.spritePath = spritePath;
```
Comment (roguefirework) [inline Core/WumpusCore/Player/Player.cs:51]: I don't think the internal wumpus core needs to know how to draw the player

File: Core/WumpusCore/Player/Player.cs [hunk 2]
```diff
@@ -7,33 +7,60 @@ public class Player
...
+        public ushort Position
+        {
+            get { return position; }
+        }
 
         /// <summary>
         /// The path of the player's sprite.
         /// </summary>
-        public string spritePath { get; private set; }
+        private string spritePath;
```
Comment (roguefirework) [inline Core/WumpusCore/Player/Player.cs:22]: Don't think we need this

**Review by roguefirework — CHANGES_REQUESTED** [review]

**Review by roguefirework — APPROVED** [review]
It indeed looks good now! YAY

**Review by MaximumMaxxx — APPROVED** [review]
Looks quite epic ngl

**Pull request merged**